### ISSUE TITLE
Mobile Devices always display the camera option regardless of ngf-capture value

### ISSUE THREAD

**Issue by ainsleyh** [danialfarid/ng-file-upload#1534]
Hello,

I am using your directive in my responsive web app. Regardless of what value I input for ngf-capture, or if I include it or not, the camera option always displays while testing on my phone. Is this a bug or am I doing something wrong?

I can't get it to work while manipulating the demo html either:
 http://angular-file-upload.appspot.com/

Thanks!

**Comment by tungtouch** [comment]
+1

**Comment by luco** [comment]
+1

**Comment by nitheeshskiploop** [comment]
+1

**Comment by Moelgaard85** [comment]
After the iOS 10.3.1 update, i got the same problem on iPad devices always going straight to camera without an option to choose library. If i remove:
```
ngf-capture="'camera'"
```
then everything works now. I get a nice native popup asking me what option to use.

**Comment by gerhardcit** [comment]
@Moelgaard85 , yes that is true for iOS, but the same code on Android (via an ionic app) does NOT allow the camera to popup with or without ngf-capture.
Have you maybe tested that on Android?

**Comment by Moelgaard85** [comment]
@gerhardcit Yes, I have tested it on Android device as well, and it works.
